pkgs/misc/uboot/default.nix

```diff
@@ -31,6 +32,7 @@ let
         url = https://github.com/dezgeg/u-boot/commit/extlinux-path-length-2018-03.patch;
         sha256 = "07jafdnxvqv8lz256qy29agjc2k1zj5ad4k28r1w5qkhwj4ixmf8";
       })
+      ./0001-rockchip-allow-loading-larger-kernels.patch
```

Comment:
has this been sent upstream? If yes, we should `fetchurl` instead.

Reply:
Better use `fetchpatch` instead of `fetchurl` in that case.